boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_defconfig

```diff
@@ -4,74 +4,19 @@ CONFIG_SOC_SERIES_NRF53X=y
 CONFIG_SOC_NRF5340_CPUAPP_QKAA=y
 CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP=y
 
-CONFIG_BOARD_ENABLE_DCDC_APP=y
-CONFIG_BOARD_ENABLE_DCDC_NET=y
-CONFIG_BOARD_ENABLE_CPUNET=y
-
 # Enable MPU
 CONFIG_ARM_MPU=y
 
-# Enable hardware stack protection
 CONFIG_HW_STACK_PROTECTION=y
 
-# Enable TrustZone-M
 CONFIG_ARM_TRUSTZONE_M=y
 
-# Enable PINCTRL
-CONFIG_PINCTRL=y
-
-# Enable GPIO
 CONFIG_GPIO=y
-CONFIG_NFCT_PINS_AS_GPIOS=y
-
-# I2S
-CONFIG_I2S=n # Zephyr I2S driver not used, we are using custom shim
-CONFIG_NRFX_I2S=y
-
-# Enable NRFX_CLOCK for ACLK control
-CONFIG_NRFX_CLOCK=y
-
-CONFIG_I2C=y
-
-# NOTE: Since we are not using minimal libc, error codes from
-#       minimal libc are not used
-CONFIG_NEWLIB_LIBC=y
-
-# Audio codec LC3 related defines
-# FPU_SHARING enables preservation of the hardware floating point registers
-# across context switches to allow multiple threads to perform concurrent
-# floating point operations.
-CONFIG_FPU=y
-CONFIG_FPU_SHARING=y
-
-# Enable SDHC interface
-CONFIG_DISK_DRIVERS=y
-CONFIG_DISK_DRIVER_SDMMC=y
-
-# Enable SPI interface
-CONFIG_SPI=y
-
-# Enable ADC for board version readback
-CONFIG_ADC=y
-
-# Allocate buffer on RAM for transferring chunck of data
-# from Flash to SPI
-CONFIG_SPI_NRFX_RAM_BUFFER_SIZE=8
-
-# Config the file system
-CONFIG_FILE_SYSTEM=y
-CONFIG_FAT_FILESYSTEM_ELM=y
-CONFIG_FS_FATFS_LFN=y
-# Using stack for LFN work queue
-CONFIG_FS_FATFS_LFN_MODE_STACK=y
-# exFAT enabled to support longer file names and higher transfer speed
-CONFIG_FS_FATFS_EXFAT=y
 
-# Set the maximum file name length to 255
-CONFIG_FS_FATFS_MAX_LFN=255
+CONFIG_SERIAL=y
 
-# Bluetooth Low Energy
-CONFIG_BT=y
+# enable console
```

Comment:
Capitial "E" for enable

Reply:
As @koffes mention.

> This needs to be done with caution as we have large boardfile changes happening in two places.

I copied defconfig straight from https://github.com/zephyrproject-rtos/zephyr/pull/49720.